src/Client/Dockerfile

```
@@ -0,0 +1,43 @@
+# 🐳 Base image with Node.js
+FROM node:23-alpine AS base
+
+# 🔧 Install dependencies
+FROM base AS deps
+RUN apk add --no-cache libc6-compat
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)